Update LVC Notices documentation, Quickstart, and announcement (#3707)
nasa-gcn/gcn.nasa.gov: app/routes/news._index/route.mdx

## 2026

<CollectionItem
    className="maxw-none grid-container"
    variantComponent={
      <CollectionCalendarDate datetime='July 22, 2026' />
    }
>

    <CollectionHeading headingLevel="h3">
      <Anchor>LVK Notices Now only Available in JSON over Kafka</Anchor>
    </CollectionHeading>
    <CollectionDescription>

>       As of July 8, 2026, the LIGO-Virgo-KAGRA (LVK) collaboration has ceased sending gravitational-wave alerts in the VOEvent format to GCN Classic. These alerts were converted to all other GCN Classic formats and distribution methods (text via email and binary via socket connection). This change also disables all legacy LVK formats distributed over GCN-Classic-over-Kafka and email.

Reviewer: ```suggestion
      As of July 8, 2026, the LIGO-Virgo-KAGRA (LVK) collaboration has ceased sending gravitational-wave alerts in the VOEvent format to GCN Classic. All other GCN Classic formats and distribution methods (text via email and binary via socket connection) for LVK were converted from these VOEvent alerts. As a result, this change also disables all legacy LVK formats distributed over GCN-Classic-over-Kafka and email.
```

>       The current deployed LVK alert system sends the hourly Mock Data Challenge (MDC) alerts only in the JSON format using GCN Kafka, and in the AVRO format using SCIMMA. The [LVK Public Alerts User Guide](https://emfollow.docs.ligo.org/userguide/) has been updated to provide information on the alert system that will be deployed in the next observation run (IR1) that is [expected to start later this year](https://observing.docs.ligo.org/plan/).

Reviewer: ```suggestion
      The current deployed LVK alert system sends the hourly Mock Data Challenge (MDC) alerts only in the JSON format using GCN Kafka, and in the AVRO format using SCIMMA. The [LVK Public Alerts User Guide](https://emfollow.docs.ligo.org/userguide/) has been updated to provide information on the alert system that will be deployed in the next observation run (IR1) that is [expected to start in late 2026](https://observing.docs.ligo.org/plan/).
```

>       The current deployed LVK alert system sends the hourly Mock Data Challenge (MDC) alerts only in the JSON format using GCN Kafka, and in the AVRO format using SCIMMA. The [LVK Public Alerts User Guide](https://emfollow.docs.ligo.org/userguide/) has been updated to provide information on the alert system that will be deployed in the next observation run (IR1) that is [expected to start later this year](https://observing.docs.ligo.org/plan/).

Reviewer: ```suggestion
      The current deployed LVK alert system sends the hourly Mock Data Challenge (MDC) alerts only in the JSON format using GCN Kafka and in the AVRO format using SCIMMA. The [LVK Public Alerts User Guide](https://emfollow.docs.ligo.org/userguide/) has been updated to provide information on the alert system that will be deployed in the next observation run (IR1) that is [expected to start later this year](https://observing.docs.ligo.org/plan/).
```

        - [Updated CHIME Notices](/news#updated-chime-notices)
        - [Schema v7.2.3](/news#schema-v723)

      #### LVK Notices Exclusively Available in JSON over Kafka
      As of July 8, 2026, the LIGO-Virgo-KAGRA (LVK) collaboration has ceased sending gravitational wave alerts in the VOEvent format to GCN Classic. The LVK VOEvent alerts were converted to all other GCN Classic formats and distribution methods (text via email and binary via socket connection). As a result, this change also disables all legacy LVK formats (text, binary, VOEvent) distributed over Kafka and email.

      GCN now distributes LVK Notices exclusively as JSON over Kafka.  The [LVK Public Alerts User Guide](https://emfollow.docs.ligo.org/userguide/) has been updated to provide information on the alert system that will be deployed in the next observation run (IR1) that is [expected to start in late 2026](https://observing.docs.ligo.org/plan/).

      To receive GCN Notices on LVK during IR1, please [subscribe to JSON-format Notices](/docs/notices/consuming) on the Kafka topic `igwn.gwalert`.  At this time, these Notices are only available via Kafka, not email.

      #### Updated CHIME Notices

>       CHIME is now distributing retractions of their FRB Notices on the topic `gcn.notices.chime.frb`, and has updated the [example](docs/schema/stable/gcn/notices/chime/frb.schema.json) added in Schema v7.2.1.

Reviewer 2: ```suggestion
      CHIME is now distributing retractions of its FRB Notices on the topic `gcn.notices.chime.frb`, and has updated the [example](docs/schema/stable/gcn/notices/chime/frb.schema.json) added in Schema v7.2.1.
```
